abm:
        fitbit:
                credentials:
                        path: '/root/state/fitbit_credentials.yaml'
                redirect_url: "http://localhost"
        state:
                path: "/root/state"
        resources:
                path: "/root/ws/catkin_ws/src/abm_interaction/resources"
